sessionmanager.md

```diff
@@ -0,0 +1,96 @@
+# `SessionManager` for @azure/service-bus v7
```

Comment:
One thing to clarify off the bat - concurrency is only being offered for working with sessions? .NET and Python (based on the proposal) are adding this generally - not just for sessions.

Reply:
This particular proposal is only for sessions, yes.

For non-session-based receiving the ability to use peekLock and the fact that users have a physical limit on the maximum number of subscriptions and queues they can create makes this less urgent.

AFAIK there's no such limit on sessions at all.

Reply:
So for the non-session receiving, you can't use parallelism?

Reply:
You can, I added an edit but let me paste this here:

https://github.com/Azure/azure-sdk-for-js/blob/47cfb37503443bfcdd16c00665214f48e0d9dd92/sdk/servicebus/service-bus/src/models.ts#L97

You can do it via `maxConcurrentCalls` today.